install/kubernetes/Makefile.digests

```diff
@@ -1,13 +1,13 @@
 # File generated by .github/workflows/build-images-releases.yaml; DO NOT EDIT.
-# Copyright Authors of Cilium
+# Copyright 2024 Authors of Cilium
```

Comment:
Hmm this is mildly interesting, we dropped dates from the copyright some time back. Not sure where this came from.